webpack.main.config.ts

```diff
@@ -1,4 +1,6 @@
 import path from 'path';
+// eslint-disable-next-line import/default
```

Comment:
We checked this error with @sejas and could not find any alternatives to fix it besides disabling the comment. I will open the issue in the repo or this plugin but since everything works as expected, we decided that it would be okay to disable it for now